src/mixins/filing-template-mixin.ts

```diff
@@ -67,6 +69,11 @@ export default class FilingTemplateMixin extends Vue {
         }
       }
     }
+    // Pass the effective date only for a future effective filing.
+    if (formattedDateTime) {
+      filing['filing']['header']['effectiveDate'] = formattedDateTime
```

Comment:
Would be better to use dot notation. I'm surprised the linter didn't complain about this.

(The named property notation you are using ignored typings/interfaces.)

Reply:
Oops.. sorry.. Corrected this